<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculater</class>
 <widget class="QMainWindow" name="Calculater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculater</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{background-color: rgb(255, 255, 255);}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:1px;
	background-color: rgb(243, 243, 243);}
QPushButton:hover{
background-color: rgb(217, 217, 217);}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:1px;
	background-color: rgb(243, 243, 243);}
QPushButton:hover{
background-color: rgb(217, 217, 217);}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:1px;
	background-color: rgb(243, 243, 243);}
QPushButton:hover{
background-color: rgb(217, 217, 217);}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:1px;
	background-color: rgb(243, 243, 243);}
QPushButton:hover{
background-color: rgb(217, 217, 217);}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_equal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:1px;background-color: rgb(213, 213, 213);}

QPushButton:hover{
background-color: rgb(218, 218, 218);}

</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:1px;
	background-color: rgb(243, 243, 243);}
QPushButton:hover{
background-color: rgb(217, 217, 217);}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:1px;
	background-color: rgb(243, 243, 243);}
QPushButton:hover{
background-color: rgb(217, 217, 217);}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:1px;
	background-color: rgb(243, 243, 243);}
QPushButton:hover{
background-color: rgb(217, 217, 217);}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:1px;
	background-color: rgb(243, 243, 243);}
QPushButton:hover{
background-color: rgb(217, 217, 217);}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:1px;
	background-color: rgb(243, 243, 243);}
QPushButton:hover{
background-color: rgb(217, 217, 217);}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:1px;
	background-color: rgb(243, 243, 243);}
QPushButton:hover{
background-color: rgb(217, 217, 217);}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:1px;
background-color: rgb(227, 227, 227);
}
QPushButton:hover{
background-color: rgb(218, 218, 218);}

</string>
    </property>
    <property name="text">
     <string>➕</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_dec">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:1px;
background-color: rgb(227, 227, 227);
}
QPushButton:hover{
background-color: rgb(218, 218, 218);}

</string>
    </property>
    <property name="text">
     <string>➖</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_multi">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:1px;
background-color: rgb(227, 227, 227);
}
QPushButton:hover{
background-color: rgb(218, 218, 218);}

</string>
    </property>
    <property name="text">
     <string>✖</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_dev">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:1px;
background-color: rgb(227, 227, 227);
}
QPushButton:hover{
background-color: rgb(218, 218, 218);}

</string>
    </property>
    <property name="text">
     <string>➗</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:1px;
background-color: rgb(227, 227, 227);
}
QPushButton:hover{
background-color: rgb(218, 218, 218);}

</string>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="result_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>12345678901234567</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="process_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(141, 141, 141);</string>
    </property>
    <property name="text">
     <string>123456789*987654321=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_square">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:1px;
background-color: rgb(227, 227, 227);
}
QPushButton:hover{
background-color: rgb(218, 218, 218);}

</string>
    </property>
    <property name="text">
     <string>x²</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_back_key">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:1px;
background-color: rgb(227, 227, 227);
}
QPushButton:hover{
background-color: rgb(218, 218, 218);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/keyboard-delete.png</normaloff>:/img/keyboard-delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>33</width>
      <height>33</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>模式(&amp;A)</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/navigation-menu.png</normaloff>:/img/navigation-menu.png</iconset>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(&amp;S)</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/network-sync.png</normaloff>:/img/network-sync.png</iconset>
    </property>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>关于(&amp;D)</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/network-question.png</normaloff>:/img/network-question.png</iconset>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>标准模式</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>进制换算模式</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>科学计算</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/network-question.png</normaloff>:/img/network-question.png</iconset>
   </property>
   <property name="text">
    <string>关于计算器beta</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset theme="清除所有" resource="img.qrc">
     <normaloff>:/img/network-sync.png</normaloff>:/img/network-sync.png</iconset>
   </property>
   <property name="text">
    <string>清除所有</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
